coc.md

```diff
@@ -0,0 +1,106 @@
+# Women Who Go - Code of Conduct
+
+GoBridge (GB) and Women Who Go (WWG) uses the Go CoC as our base document:
+
+[Go CoC](https://golang.org/conduct) | [Go Forum](https://forum.golangbridge.org/) | [Gophers Slack](http://invite.slack.golangbridge.org) | [Women Who Go Slack](https://womenwhogo.org/invite.html)
+
+## GB/WWG Addendum
+
+### Real Names
+If you feel comfortable, we encourage you to use your real name in all of our online and offline GB/WWG communities and events. We find it encourages openness and civility in discussions. However, this is not a requirement.
+
+Organizers and moderators may ask people to change their handles or usernames if it conflicts with known public figures, impersonates other people, sounds like automated tools, contains inappropriate language, or other unclear cases.
+
+### Language Policy
+We are working hard to keep our public spaces family-friendly and as inclusive as possible. At the organizers or moderators discretion, words and images that are deemed inappropriate or offensive will be moderated. Generally, cursing is not allowed. Please reach out to the organizers or moderators for a list of words that are being moderated.
+
+### Moderation
+Participants in our online and offline GB/WWG communities and events should flag inappropriate behavior to organizers and moderators rather than addressing it themselves. This includes but is not limited to reporting: violations of this code of conduct, spam messages, messages posted in the wrong channel or category, and public conversations that are inappropriate. Please report any violations to the #admin-help channel on gophers.slack.com as early as possible or privately message Carmen Andoh or William Kennedy directly. At our offline events please report any violations to the organizers or email support@gobridge.org.
```

Comment:
I have just added a channel named #admin-help to our WWG slack so that we can more easily have consistent instructions across our slacks.

I also added my name to the list of people to whom you can report a CoC concern for WWG.

```suggestion
Participants in our online and offline GB/WWG communities and events should flag inappropriate behavior to organizers and moderators rather than addressing it themselves. This includes but is not limited to reporting: violations of this code of conduct, spam messages, messages posted in the wrong channel or category, and public conversations that are inappropriate. Please report any violations to the #admin-help channel slack as early as possible or privately message Carolyn Van Slyck directly. At our offline events please report any violations to the organizers or email hello@womenwhogo.org.
```